bentoml/_internal/cli/bento_server.py

```diff
@@ -13,7 +14,7 @@
 
 
 def add_serve_command(cli: click.Group) -> None:
-    @cli.command()
+    @cli.command(help="Serve a Bento")
```

Comment:
I think with `click`, this actually will override the docstring below in the `--help` message, is that correct?  if so, let's remove it

Reply:
will do